Formula/p/poac.rb

```diff
@@ -16,6 +16,9 @@ class Poac < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "902689bc9c40c5d9ac2c51fa4006f45ad7703a8c3f1cec033d2f0fd4d9c6012b"
   end
 
+  depends_on "libgit2"
+  depends_on "pkg-config"
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "libgit2"
```

Reply:
@p-linnane pkg-config is also a runtime dependency, but do we need to make this change?

https://github.com/poac-dev/poac/tree/0.8.0?tab=readme-ov-file#runtime-requirements

Reply:
My apologies. I don't believe we have any existing packages that need `peg-config` at runtime, so I assumed that was an error. Thanks for letting me know.

Reply:
No worries! Thank you for your review!

Reply:
I stand corrected! Turns out `poac` will be the 17th formulae with a `pkg-config` runtime dependency.